packages/jest-diff/src/cli/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import yargs from 'yargs';
+import {readFile} from 'fs';
+
+// eslint-disable-next-line import/default
+import jestDiff from '../';
+import {NO_DIFF_MESSAGE} from '../constants';
+
+/* eslint-disable sort-keys */
+const EXIT_CODES = {
+  // like GNU diff
+  EQUAL: 0,
+  DIFFERENT: 1,
+  INVALID_USAGE: 2,
+
+  // more specific than GNU diff, which exits with 2 in all error cases
+  INTERNAL_ERROR: 10,
+  IO_ERROR: 20,
+  INVALID_JSON: 30,
+};
+/* eslint-enable sort-keys */
+
+const read = (path: string) =>
+  new Promise(resolve =>
+    readFile(path, (readErr, data) => {
+      if (readErr) {
+        console.error(`Failed to read file ${path}`, readErr);
+        process.exit(EXIT_CODES.IO_ERROR);
+      }
+      try {
+        const json = JSON.parse(String(data));
+        resolve(json);
+      } catch (parseErr) {
+        console.error(`Failed to parse file ${path} as JSON`, parseErr);
+        process.exit(EXIT_CODES.INVALID_JSON);
+      }
+    }),
+  );
+
+export default async () => {
+  const parser = yargs(process.argv.slice(2))
+    // positional
+    .demandCommand(2, 2)
+    // version
```

Comment:
unnecessary comment
```suggestion
```

Reply:
Can remove them, but I'll leave empty lines in place to keep things grouped up. These yargs builders can get pretty long over time :smile: